Formula/lmdb.rb

```diff
@@ -16,20 +15,15 @@ class Lmdb < Formula
   end
 
   def install
-    inreplace "libraries/liblmdb/Makefile" do |s|
-      s.gsub! ".so", ".dylib"
-      s.gsub! "$(DESTDIR)$(prefix)/man/man1", "$(DESTDIR)$(prefix)/share/man/man1"
+    cd "libraries/liblmdb" do
+      inreplace "Makefile", ".so", ".dylib"
```

Comment:
This is upstream's fix: https://github.com/LMDB/lmdb/commit/c6510147cee88d182af431c14a61fad6fae18be3

I'm not sure that's what you two had in mind.